test/unix/force_merge.ml

```diff
@@ -338,6 +338,28 @@ let merge_during_sync () =
   Index.close rw;
   Index.close ro
 
+let test_is_merging () =
+  let Context.{ rw; _ } = Context.empty_index () in
+  let f msg b () =
+    Log.debug (fun l -> l "test ");
+    Alcotest.(check bool) msg (Index.is_merging rw) b
+  in
+  f "before merge" false ();
+  let k1, v1 = (Key.v (), Value.v ()) in
+  Index.replace rw k1 v1;
+  let t = Index.force_merge ~hook:(before (f "before" true)) rw in
+  Index.await t |> check_completed;
```

Comment:
This could be a helper e.g. `add_binding_and_merge ~hook`, to be used 3 times.

Reply:
It's nicer with your suggestion, thanks!